settingslogic-gh.gemspec

```diff
@@ -2,12 +2,12 @@
 $:.push File.expand_path("../lib", __FILE__)
 
 Gem::Specification.new do |s|
-  s.name        = "settingslogic"
-  s.version     = "2.0.9"
+  s.name        = "settingslogic-gh"
+  s.version     = "2.0.9.gh.1"
```

Comment:
Does this need to be `2.0.9.1` or something to upgrade properly from `2.0.9`, or is this okay as is?

Reply:
Since the gem name is changing, and we're technically using a pre-release version, I don't think it will be n upgrade so much as a replacement. I assumed we'd put it into our service gemfiles as `gem  'settingslogic-gh', '~> 2.0.9.gh.1'`, and if we release an update `2.0.9.gh.2` would be considered an upgrade by bundler.

Reply:
Oh yeah, fair enough.